webui/react/src/pages/Settings/UserManagement.tsx

```diff
@@ -104,44 +106,40 @@ const UserActionDropdown = ({ fetchUsers, user, groups }: DropdownProps) => {
 };
 
 const UserManagement: React.FC = () => {
-  const [users, setUsers] = useState<DetailedUser[]>([]);
   const [groups, setGroups] = useState<V1GroupSearchResult[]>([]);
-  const [isLoading, setIsLoading] = useState(true);
-  const [total, setTotal] = useState(0);
   const [canceler] = useState(new AbortController());
   const pageRef = useRef<HTMLElement>(null);
-
+  const fetchUsersHook = useFetchUsers(canceler);
   const { settings, updateSettings } = useSettings<UserManagementSettings>(settingsConfig);
+  const apiConfig = useMemo<FetchUsersConfig>(
```

Comment:
bug of sorting
sorting is not correct

https://user-images.githubusercontent.com/109113616/210413857-66677dc2-ffa0-4795-ba4d-745e4e41ab26.mov